### ISSUE TITLE
Error when attempting to use attic data

### ISSUE THREAD

**Issue by DavidMoraisFerreira** [wiktorn/Overpass-API#5]
Hi,
For some reason, I can't get get queries to work that use the date (attic) feature.

For `wget -O - "http://192.168.1.147:12345/api/interpreter?data=[date:\"2017-10-28T19:20:00Z\"];node[\"addr:housenumber\"];out;"`

I get the following result/error:

```
<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API 0.7.55 579b1eec">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base="2018-04-28T07:13:02Z"/>
<remark> runtime error: open64: 2 No such file or directory /db/db/node_tags_global_attic.bin File_Blocks::File_Blocks::1 </remark>
</osm>
```
By removing `[date:\"2017-10-28T19:20:00Z\"];` from the query, OSM data is returned.

Could someone have a look at this issue?
Thank you

#### Steps to reproduce:
I followed the README example to start an overpass-api instance for Monaco with **OVERPASS_META=attic**.
I also tried **OVERPASS_META=yes**, but it made no difference.

```
docker run \
  -e OVERPASS_META=attic \
  -e OVERPASS_MODE=init \
  -e OVERPASS_PLANET_URL=http://download.geofabrik.de/europe/monaco-latest.osm.bz2 \
  -e OVERPASS_DIFF_URL=http://download.openstreetmap.fr/replication/europe/monaco/minute/ \
  -e OVERPASS_PLANET_SEQUENCE_ID=2946200 \
  -e OVERPASS_RULES_LOAD=10 \
  -v /data/docker/overpass_db/:/db \
  -p 12345:80 \
  -i -t \
  --name overpass_monaco wiktorn/overpass-api

docker start overpass_monaco

wget -O - "http://192.168.1.147:12345/api/interpreter?data=[date:\"2017-10-28T19:20:00Z\"];node[\"addr:housenumber\"];out;"

wget -O - "http://192.168.1.147:12345/api/interpreter?data=node[\"addr:housenumber\"];out;"


```

**Comment by DavidMoraisFerreira** [comment]
Even if the aforementioned example only dates to April 2018, the binary files (/db/db/node_tags_global_attic.bin) required to query the "attic data" aren't included in the docker image. Or is `OVERPASS_META=attic` only supported with `OVERPASS_MODE=clone`?

**Comment by wiktorn** [comment]
There was a bug that was fixed in d442d56 - once the docker images rebuild on Docker Hub it should start to work
